spyder/plugins/variableexplorer/widgets/dataframeeditor.py

```diff
@@ -91,6 +91,18 @@
 BACKGROUND_MISC_ALPHA = 0.3
 
 
+def is_any_real_numeric_dtype(dtype) -> bool:
+    """
+    Test whether a Pandas dtype is a real numeric type.
+    """
+    try:
+        import pandas.api.types
+        return pandas.api.types.is_any_real_numeric_dtype(dtype)
+    except AttributeError:
```

Comment:
```suggestion
    except Exception:
```

Just in case the import fails for any reason.